crates/cli-util/src/opts.rs

```diff
@@ -72,6 +76,24 @@ pub(crate) struct NetworkOpts {
     pub request_timeout: u64,
 }
 
+impl CommonClientConnectionOptions for NetworkOpts {
+    fn connect_timeout(&self) -> Duration {
+        Duration::from_millis(self.connect_timeout)
+    }
+
+    fn keep_alive_interval(&self) -> Duration {
+        Duration::from_secs(60)
```

Comment:
Hard-coding this for the CLI should be okay; wanted to keep the trait as-is to avoid changing prod code.